scripts/postman_test_automation.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# [ DECLARATIONS ]
+KEY_TYPE=""
+
+API_KEY=""
+API_SECRET=""
+KEY1=""
+
+# Unused as of now, will be useful once we start using this properly
+CERTIFICATE=$4
+CERTIFICATE_KEY=$5
+
+path_generation() {
+    local NAME=$1
```

Comment:
1. Local variables can be `snake_case`. I personally feel it makes it easier to identify variables which have a global scope against local ones.
2. Always quote the usages of variables/arguments.

```suggestion
    local name="$1"
```

Optionally, you can also prefer to wrap them within `{}` like so:

```suggestion
    local name="${1}"
```

So that a variable substitution like `"$12"` do not unintentionally search for the 12th argument when you instead wanted to suffix a `2` to the value of the 1st argument: `"${1}2"`. You can read more about this.